src/lib/phrases/infrastructure/components/phrase.svelte

```diff
@@ -5,9 +5,11 @@
   import relativeTime from "dayjs/plugin/relativeTime";
   dayjs.extend(relativeTime);
 
+  export let id: bigint;
```

Comment:
**suggestion:** remove needless id parameter

As the id parameter is only used in the onClose method, and both are given by the parent. It doesn't make much sense for this component to use them. It seems that the parent is the correct one to handle the close by id instead.